src/main/java/uk/gov/justice/digital/delius/transformers/OffenderTransformer.java

```diff
@@ -268,15 +270,17 @@ private static Team teamOf(uk.gov.justice.digital.delius.jpa.standard.entity.Off
          * This only populates a subset of team data - this is currently indexed in elasticsearch so if this is modified
          * then it will lead to inconsistent data and the index will need to be rebuilt.
          */
-        return Optional.ofNullable(offenderManager.getTrustProviderTeam())
+        return Optional.ofNullable(offenderManager.getTeam())
```

Comment:
Change from ALL_TEAMS table to TEAMS table to be consistent with the offender manager endpoint. ALL_TEAMS contains everything from TEAMS plus a handful of pre CRC teams that were all closed 7 years ago and have no OMs related to them